### DISCUSSION TITLE
Переместить кнопку

### DISCUSSION THREAD

**Discussion by JustUserFromMars** [ilyhalight/voice-over-translation#407] [Q&A] [answered]
Можно добавить кнопку перемещения расширения по экрану? Или подскажи что где в Tampermonkey поправить. чтобы скажем в угол экрана переместить. Спасибо
![image](https://github.com/ilyhalight/voice-over-translation/assets/150505742/9b586e9a-966b-49a8-bba0-d3e83e447560)

**Comment by JustUserFromMars** [comment]
АлооО.

**Comment by ilyhalight** [comment, marked as answer]
Чтобы перемести к углу экрана редактируйте стиль `left` для `.vot-segmented-button`. Перемещение меню не планируется
